app/src/main/res/values-en/strings.xml

```diff
@@ -1236,4 +1236,154 @@
     <string name="permissions_denied">You denied the required permissions for the application.\n\nIn order to grant the permission, open the Permissions screen for Szkolny.eu in phone settings.\n\nClick OK to open app settings now.</string>
     <string name="permissions_required">Required permissions</string>
     <string name="settings_register_hide_sticks_from_old">Your mother won\'t see your F grades</string>
+    <string name="build_official">Official build</string>
+    <string name="build_platform_play">Google Play</string>
+    <string name="login_chooser_testing_title">Test version</string>
+    <string name="login_chooser_mode_recommended">{cmd-information-outline} Recommended</string>
+    <string name="login_chooser_mode_dev_only">{cmd-android-studio} Developer version</string>
+    <string name="login_chooser_mode_testing">{cmd-alert-circle-outline} Test version</string>
+    <string name="eggs">\???</string>
+    <string name="see_also">See also</string>
+    <string name="settings_about_github_text">Source code</string>
+    <string name="build_branch">Git branch</string>
+    <string name="build_platform_apk">.APK</string>
+    <string name="build_platform_unofficial">Unofficial (.APK)</string>
+    <string name="build_commit">Commit hash</string>
+    <string name="build_dirty">Unsaved changes</string>
+    <string name="build_tag">Last build tag</string>
+    <string name="build_rev_count">Commits from last build tag</string>
+    <string name="build_remote">Remote repository</string>
+    <string name="build_details">Build details</string>
+    <string name="build_dialog_open_repo">Check code</string>
+    <string name="profile_config_name_hint">Profile name</string>
+    <string name="profile_config_logout">Logout</string>
+    <string name="profile_config_sync_enabled">Synchronize that profile</string>
```

Comment:
As the option is present in the dialog of a specified profile, `this` would be more appropriate than `that`.